docs/quickstart/README.md

```diff
@@ -0,0 +1,298 @@
+                                                             tl.tileDimensions._2))) 
+    }
+```
+
+Finally, it is necessary to combine the results.  Note that, in order to produce a 700x400 tile that is identical to the simpler, non-tiled case, every SpatialKey must be represented in the map, or the result may not span the full range.  This is only necessary if it is important to generate a tile that spans the full extent.
+
+```scala
+  import geotrellis.spark.stitch.TileLayoutStitcher
+
+  val tileList = 
+      for { r <- 0 until ld.layoutRows
+          ; c <- 0 until ld.layoutCols
+          } yield {
+            val k = SpatialKey(c,r)
+            (k, keytiles.getOrElse(k, IntArrayTile.empty(tl.tileCols, 
+                                                         tl.tileRows)))
+          }
+    val stitched = TileLayoutStitcher.stitch(tileList)._1
+```
+
+It is now the case that `stitched` is identical to `kde.`
+
+### Distributing Computation via Apache Spark ###
+
+As mentioned, the reason for introducing this more complicated version of kernel density estimation was to enable distributed processing of the kernel stamping process.  Each subtile could potentially be handled by a different node in a cluster, which would make sense if the dataset in question were, say, the location of individual trees, requiring a huge amount of working memory.  By breaking the domain into smaller pieces, we can exploit the distributed framework provided by Apache Spark to spread the task to a number of machines.  This will also provide fault tolerant, rapid data processing for real-time and/or web-based applications.  Spark is much too big a topic to discuss in any detail here, so the curious reader is advised to search the web for more information.
+
+Our concern falls on how to write code to exploit the structures provided by Spark, specifically *Resilient Distributed Datasets*, or RDDs.  An RDD is a distributed collection, with all of the usual features of a collection---i.e., mapping, folding, reducing.  In order to create an RDD, one will call the `parallelize()` method on a `SparkContext` object, which can be generated by the following set of statements.
+```scala
+  import org.apache.spark.{SparkConf, SparkContext}
+  val conf = new SparkConf().setMaster("local").setAppName("Kernel Density")
+  val sc = new SparkContext(conf)
+```
+
+In our case, we have a collection of PointFeatures that we wish to parallelize, so we issue the command
+```scala
+  import org.apache.spark.rdd.RDD
+  val pointRdd = sc.parallelize(pts, 10)
+```
+Here, the 10 indicates that we want to distribute the data in 10 pieces to the available workers.  Some of those chunks may end up on different physical machines if they are available.  If we exclude this value, the default parallelism will be used, which is typically the number of processors, though in this local example, it defaults to one.
```

Comment:
pieces -> partitions. Might was well use the verbiage outright. Would be worth it to quickly explain, `a partition is a subset of the data in the RDD that will be processed by one of the workers, where partitions can be processed in parallel over all workers`.